### DISCUSSION TITLE
Install new solis inversor

### DISCUSSION THREAD

**Discussion by kiloptero** [Pho3niX90/solis_modbus#33] [Q&A] [answered]
For a new solis inverter, what would be the process? We have to connect the datalogger to the wifi and then via the integration to the ip dongle nothing else? do we have to configure something in the inverter or first connect it to solis cloud? I don't want to make a mistake and then not be able to use it. thanks a lot.

**Comment by Pho3niX90** [comment, marked as answer]
There is no extra setup required on the inverter side. Just simple datalogger to wifi, and then set the ip in the plugin as you said.

**Reply by kiloptero** [reply]
thanks
